### PR TITLE
Task4255_Update_test_site_addFeaturesToBottomSheet

### PR DESCRIPTION
**Pull request by MayaElf**
fullscreen
 hide-overlay
 width/max-width
 overlay-color
 overlay-opacity
 scrollable

### CODE REVIEW

**Comment by MayaElf** [comment]
Sorry, dublicate by mistake

**Comment by MayaElf** [comment]
Sorry, I forgot to change the name of the commit :(

File: components/bottom_sheets/ModelBottomSheet.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
     <div class="text-center">
         <v-btn color="blue" dark @click="sheet = !sheet"> Open v-model </v-btn>
-        <v-bottom-sheet v-model="sheet">
-            <v-sheet class="text-center" height="200px">
+        <v-bottom-sheet v-model="sheet" fullscreen>
+            <v-sheet class="text-center" height="800px">
```
Comment (omega790) [inline components/bottom_sheets/ModelBottomSheet.vue:5]: height="800px" -> height="100%"

File: components/bottom_sheets/PersistentBottomSheet.vue [hunk 2]
```diff
@@ -1,6 +1,13 @@
 <template>
     <div class="text-center">
-        <v-bottom-sheet v-model="sheet" persistent>
+        <v-bottom-sheet
+            v-model="sheet"
+            persistent
+            overlay-color="purple"
+            overlay-opacity="0.1"
+            width = 500
+            no-click-animation
```
Comment (omega790) [inline components/bottom_sheets/PersistentBottomSheet.vue:9]: Could you tell me please what property to check that on site?

Comment (MayaElf) [reply components/bottom_sheets/PersistentBottomSheet.vue:9]: It is **Persistent bottom sheet**

File: components/bottom_sheets/ModelBottomSheet.vue (hunk 1 above)
Comment (MayaElf) [reply components/bottom_sheets/ModelBottomSheet.vue]: DONE :)

File: components/bottom_sheets/PersistentBottomSheet.vue (hunk 2 above)
Comment (omega790) [reply components/bottom_sheets/PersistentBottomSheet.vue:9]: I am about "no-click-animation"

Comment (MayaElf) [reply components/bottom_sheets/PersistentBottomSheet.vue:9]: Oh, I see. When we click outside bottomsheet the layout 'bounces', with 'no-click-animation' feature it stands still.

Comment (MayaElf) [reply components/bottom_sheets/PersistentBottomSheet.vue]: feature 'no-click-animation' - not testable - deletion approved by @omega790 (info in matrix)

**Review by omega790 — APPROVED** [review]

**Pull request merged**